### ISSUE TITLE
redefine disease entity in  Mondo FAQ

### ISSUE THREAD

**Issue by nicolevasilevsky** [monarch-initiative/mondo#6593]
please edit the text below as needed, this will go on the [FAQ](https://mondo.monarchinitiative.org/pages/faq/)
per our discussion on the Mondo medical expert call on 2023-08-24

---
### What is the difference between a disease entity and a phenotype?

There is a lot of overlap between diseases and phenotypes. Mondo and Human Phenotype Ontology (HPO) define a _disease entity_ (ie not a group of diseases) as an entity that has the four following features:

**A disease entity has:**   
1. an etiology (whether identified or as yet unknown)
2. phenotypic feature(s)
  - often, a constellation of multiple phenotypic features, each of which can have a different onset.
  - in many cases, the phenotypic features can be observed in many diseases and thus can have multiple etiologies (presumably representing the final common pathophysiological pathway). For instance, HP:0001337 Tremor can be observed in many diseases with multiple causes.
  - in the HPO, phenotypic features usually do not encode time courses but represent a single observation. This is difficult for features that can occur repeatedly (HP:0001250 Seizure)
  - a phenotypic feature usually should not describe more than one feature (HPO leaf term), i.e., it should be atomic. The HPO has some terms such as “Schizophrenia” that are used frequently in the literature to record a set of features and may characterize other diseases.
3. a characteristic natural history or time course (progressive or static).

**A disease entity may have:**  
4. a typical response to treatment(s)

Note, that this definition applies to **leaf nodes**. For instance, Parkinson’s Disease is NOT a disease entity; it is a grouping term because it has multiple children in Mondo, such as [MONDO:0011613 'autosomal recessive early-onset Parkinson disease 6'](https://www.ebi.ac.uk/ols4/ontologies/mondo/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FMONDO_0011613), which is a disease entity (leaf term in Mondo). Current efforts are underway to separate disease entities from disease classes or groupings in Mondo.

A phenotype (or a phenotypic feature) is a component of a disease entity, and HPO terms can be used to describe the set of phenotypic features that characterize a disease entity. There is a grey zone between diseases and phenotypic features. For instance, diabetes mellitus can be conceptualized as a disease, but it is also a feature of other diseases such as Bardet Biedl syndrome. In some cases like this, Mondo may have an OBO Foundry unique label that is the disease entity with (disease) after the disease name, to distinguish it from the HPO phenotype term (for example, [MONDO:0005466 hypersomnia](https://www.ebi.ac.uk/ols4/ontologies/mondo/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FMONDO_0005466) and [HP:0100786 Hypersomnia](https://hpo.jax.org/app/browse/term/HP:0100786)).

**Comment by nicolevasilevsky** [comment]
I updated the FAQ page. We can revise further if needed

**Comment by cmungall** [comment]
I'm afraid the documentation is incorrect and misleading.

"For instance, Parkinson’s Disease is NOT a disease entity; it is a grouping term because it has multiple children in Mondo"

Leaf nodes are not the same as disease entities! Leafness is just a property of the graph!

**Comment by pnrobinson** [comment]
Currently there are multiple problems

1. Parkinson disease is-a hereditary neurological disease (this is not always or even usually true)
2. The main axis of classification is early onset and late onset. However, this seems to be comparing the hereditary forms of parkinson with each other. For instance, Parkinson 1 is listed as late onset in Mondo but it is described as " Onset mid to late adulthood" in OMIM. Compared with non-hereditary Parkinson, an onset in mid-adulthood cannot be described as "late"
3. Most importantly, there is no way of denoting a specifically non-Mendelian form of Parkinson. Chris seems to be implying that the inner node "Parkinson disease" might have this meaning, but I disagree. The Non-hereditary form of Parkinson should NOT have all of the hereditary forms of Parkinson as children. This type of Parkinson is called simply "Parkinson disease" in much literature, but actually the name should be something like "Sporadic Parkinson disease (PD)" or "Idiopathic PD", or even "Non-Mendelian PD".

This is a problem in many other places of Mondo. It would be good to write down a strategy for this -- the current documentation is very geek heavy and is very inadequate for actually how to curate cases such as this!

**Comment by cmungall** [comment]
My my main point here is that we should not equate being a leaf node with being a disease entity, the documentation just happened to have PD!
